lib/init/helptext.html

```diff
@@ -14,160 +14,135 @@ <h2>1. Graphical startup of GRASS GIS</h2>
 
 <i>For text-based startup see below.</i>
 <p>
-
-After <a href="grass7.html">launching</a> GRASS GIS, the startup screen will open:
+Compared to previous versions of GRASS, the concept of starting has completely
+changed in version 8. You will no longer find the graphical startup screen in
```

Comment:
Let's remove this paragraph, it will be confusing for new users, we don't need to talk about the past state.